server/src/bots/telegram/botResponse/runInterruptedCommandReponse.ts

```diff
@@ -0,0 +1,19 @@
+import * as TelegramBot from 'node-telegram-bot-api';
```

Comment:
@danbilokha what is interrupted command?

Reply:
discussed